src/ngAria/aria.js

```diff
@@ -377,6 +377,25 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
             });
           }
         }
+
+        if (elem[0].nodeName === 'A') {
+          if ($aria.config('bindRoleForClick') && !attr.href && !attr.xlinkHref && !attr.role) {
+              elem.attr('role', 'link');
+          }
+          if ($aria.config('bindKeypress')) {
+            elem.on('keypress', function(event) {
+              var keyCode = event.which || event.keyCode;
+              var hasHref = (attr.href || attr.xlinkHref);
+              if ((keyCode === 32 || (keyCode === 13 && !hasHref)) && !attr.ngKeypress) {
+                elem[0].click();
```

Comment:
I was aiming to make the behavior the same for ```<a>``` elements across the varying types of input. So regardless if the user clicks with the mouse, or uses the keyboard or screen reader to press SPACE or ENTER, the behavior should be the same. If someone is using ```fn``` on anchor tags with the ```hrefs``` defined, then the two keys behave differently - one follows the link, while the other executes the ```ng-click``` and stops. Using ```.click``` resolves this inconsistency.  I guess I leave it to you to tell me if this is a scenario we are not concerned with. 

Perhaps as another feature enhancement, ngAria should be applying behavior to ```ng-href``` to ensure anchor keyboard input is consistent for SPACE and ENTER.